specs/interop/shared-locbox.md

```diff
@@ -0,0 +1,300 @@
+# Shared Lockbox
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Implementation](#implementation)
+  - [`SharedLockbox`](#sharedlockbox)
+  - [`SuperchainConfig`](#superchainconfig)
+  - [`OptimismPortal`](#optimismportal)
+  - [`LiquidityMigrator`](#liquiditymigrator)
+  - [Future Considerations / Additional Notes](#future-considerations--additional-notes)
+- [Upgrade and migration process](#upgrade-and-migration-process)
+  - [Batch transaction process](#batch-transaction-process)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+With interoperable ETH, withdrawals may fail if the referenced `OptimismPortal` lacks sufficient ETH
+— making it mandatory to withdraw from the L2 where the ETH was originally deposited.
+To improve the Superchain's interoperable ETH withdrawal user experience and avoid this issue,
+the `SharedLockbox` serves as a solution.
+It unifies ETH L1 liquidity in a single contract, enabling seamless withdrawals of ETH from any OP chain in the Superchain,
+regardless of where the ETH was initially deposited.
+
+## Implementation
```

Comment:
Is implementation the right word for this section? Specs are usually neutral regarding implementation. Maybe Design/properties?